modules/helm/format.go

```diff
@@ -0,0 +1,81 @@
+package helm
+
+import (
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	"github.com/gruntwork-io/gruntwork-cli/collections"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
```

Comment:
Should terratest depend on `gruntwork-cli`?

Reply:
Well both of these are useful libraries in tests and main code and I don't think they change that much between tests and prod. Honestly, I look at `gruntwork-cli` more as a Gruntwork utils stdlib and less as a "production CLI lib" given some of the modules it provides.

e.g The stacktrace errors functionality is something that is very useful in test code.

I am open to suggestions. We can reimplement these functions in terratest (cringe), or we can create a new gruntwork-stdlib or something like that.

Reply:
No problem, we can keep using it, I was just checking if it was intentional.